public/app/features/dashboard/services/DashboardSrv.ts

```diff
@@ -43,6 +61,18 @@ export class DashboardSrv {
     });
   }
 
+  saveDashboard(
+    data: SaveDashboardOptions,
+    requestOptions?: Pick<BackendSrvRequest, 'showErrorAlert' | 'showSuccessAlert'>
+  ) {
+    return getBackendSrv().request({
```

Comment:
`request` is deprecated. I think you can use `.post` here?

Reply:
i forgot to push the last commit, sorry!

i can't use `post` because i also need for my usecase to pass in extra `BackendSrvRequest`  (in my case `showErrorAlert` and `showSuccessAlert`)